packages/angular-cli/lib/config/schema.json

```diff
@@ -135,10 +135,13 @@
         },
         "prefixInterfaces": {
           "type": "boolean"
+        },
+        "poll": {
+          "type": "number"
```

Comment:
`null` should be the default value, that is the default value for poll in watchpack. This will allow for no change to current Angular-CLI users, and make polling opt-in only.